calculator/src/button.js

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
+import './button.css';
+const Button =() => {
+//  const [number,setNumber] = useState("");
+ const [result,setResult] = useState("test");
+
+ const handleClick = (e) => {
+     setResult(result.concat(e.target.value))
+ }
+ const clearDisplay = ()=> {
+     setResult("");
+ }
+ const calculateResult =() => {
+     setResult(eval(result));
+ }
```

Comment:
It is better to add a new line before return statement (for better readability).